react.js

```diff
@@ -35,33 +58,40 @@ this.state = {
   count: 0
 }
 
```

Comment:
You have the functionality, but it should be wrapped in a method so count can be updated when the method is called.